working_notebook-TO_COMPLETE_Anna.ipynb

```diff
@@ -815,19 +815,25 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.                value_counter = 40
</pre><p>Beware that this init is performed for <em>every</em> value.. so <code>value_counter</code> is always <code>39</code> at the end of the loop :P</p><p> </p><p>This is why, <strong>indentation is important</strong> ;)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WebValley2021ReImagined/programming-for-data-science/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>